src/modules/intelligence/domain/entities/analyses_content.py

```
@@ -0,0 +1,40 @@
+    created_at: datetime
+    updated_at: datetime
+
+    def __init__(
+        self,
+        analysis_id: UUID,
+        language: str,
+        summary: Optional[str] = None,
+        pain_points: Optional[str] = None,
+        insights: Optional[str] = None,
+        innovations: Optional[str] = None,
+        id: Optional[UUID] = None,
```

Ruff (0.15.12)

[error] Function argument `id` is shadowing a Python builtin

(A002)